mirrord/layer/src/file/hooks.rs

```diff
@@ -1212,7 +1212,14 @@ pub(crate) unsafe fn enable_file_hooks(hook_manager: &mut HookManager) {
             FN___LXSTAT64
         );
         replace!(hook_manager, "lstat", lstat_detour, FnLstat, FN_LSTAT);
-        replace!(hook_manager, "fstat", fstat_detour, FnFstat, FN_FSTAT);
+        crate::replace_with_fallback!(
```

Comment:
Import it instead of `crate::`

Reply:
for a one time?

Reply:
It's fine either way, just that the others we import.

In the end, doesn't really matter, up to you.